documentation/book/ref-kafka-authentication.adoc

```diff
@@ -15,6 +15,9 @@ Supported authentication mechanisms:
 
 * TLS client authentication
 * SASL SCRAM-SHA-512
+* xref:assembly-oauth-str[{oauth} token based authentication]
+
```

Comment:
Do the 3 empty lines have some special meaning in terms of the resulting formating?

Reply:
Removed